src/docs/common_tasks/multiple_jvm_versions.md

```diff
@@ -0,0 +1,53 @@
+
+## Problem
+
+You want to compile and/or run different test or binary targets using different JVMs.
+
+Or, you need to specify command-line options globally for the JVM on a per-JVM version basis. For
+example, using `--add-opens` flags, but only when running on JVMs that support it.
+
+## Solution
+
+Declare JVM platforms using the `jvm-platform` configuration section to define the set of available
+platforms.
+
+On targets, add the kwarg `runtime_platform` set to the name of one of the platforms defined in
+`jvm-platform`. That will tell pants to use that platform when running that target in a JVM.
+
+To compile with a specific platform, set the `platform` on the target.
+
+
+## Discussion
+
+For compile time configuration, there are three main attributes, `source`, `target` and `args`.
+`source` and `target` map directly to the javac arguments of the same name, though they accept
+ more aliases than javac. `args`, allows the platform to specify additional, global to that
```

Comment:
> [...] more aliases than javac.

Examples here would be good, or even better: a pointer to a full list of aliases.